### PR TITLE
Add Photo Management Recommendations

### PR DESCRIPTION
**Pull request by jonaharagon**
**Important conflict of interest update, see:** https://github.com/privacyguides/privacyguides.org/pull/2102#issuecomment-1747401016

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/en/code_of_conduct/).

### CODE REVIEW

**Comment by nep2ner** [comment]
It will be awesome to add this category to privacy guides. [Immich](https://immich.app) (self host) should be included for evaluation as well!

**Comment by dngray** [comment]
Might be worth mentioning this https://ente.io/blog/cryptography-audit/ in the ente description now they've done that.

File: docs/photo-management.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    [:octicons-eye-16:](https://ente.io/privacy){ .card-link title="Privacy Policy" }
+    [:octicons-info-16:](https://ente.io/faq){ .card-link title=Documentation}
+    [:octicons-code-16:](https://github.com/ente-io){ .card-link title="Source Code" }
+
+    ??? downloads
+
+        - [:simple-googleplay: Google Play](https://play.google.com/store/apps/details?id=io.ente.photos)
+        - [:simple-android: Android](https://ente.io/download)
+        - [:simple-appstore: App Store](https://apps.apple.com/app/id1542026904)
+        - [:simple-github: GitHub](https://github.com/laurent22/joplin-android/releases)
```
Comment (vishnukvmd) [inline docs/photo-management.md:22]: Hey @jonaharagon, could you please update this link? We offer releases over Github for Android @ https://github.com/ente-io/photos-app/releases, and Desktop @ https://github.com/ente-io/photos-desktop/releases

Comment (vishnukvmd) [reply docs/photo-management.md:22]: We're also available on F-Droid @ https://f-droid.org/packages/io.ente.photos.fdroid/

Comment (dngray) [reply docs/photo-management.md:22]: Oops someone made a copy paste error.

**Comment by vishnukvmd** [comment]
Hey @jonaharagon, @dngray, any chance we can merge this PR? It's been a while since we've been approved. I understand you guys are dealing with a ton of work, and it'd be great if we could move forward.

**Comment by ua741** [comment]
For Cloud based photo management products, that can not be self-hosted, we can borrow points from `Cloud Storage` section.

[Minimum Requirements](https://www.privacyguides.org/en/cloud/#minimum-requirements)

- Must enforce end-to-end encryption.
- Must offer a free plan or trial period for testing.
- Must support TOTP or FIDO2 multi-factor authentication, or Passkey logins.
- Must offer a web interface which supports basic file management functionality.
- Must allow for easy exports of all files/documents.
- Must use standard, audited encryption.

**Comment by jonaharagon** [comment]
I'm recusing myself from this discussion/vote, because ente photos [sponsors](https://discuss.techlore.tech/t/welcoming-our-first-forum-sponsor-ente-open-source-e2ee-photo-storage/5482) my employer as of Sept 2023. Some other team member will have to take over getting 2 team approvals other than myself and merging this PR.

File: docs/tools.md [hunk 2]
```diff
@@ -170,6 +170,16 @@ We [recommend](dns.md#recommended-providers) a number of encrypted DNS servers b
...
+### Photo management
+
+<div class="grid cards" markdown>
+
+- ![Ente logo](assets/img/photo-management/ente.svg#only-light){ .twemoji }![Ente logo](assets/img/photo-management/ente.svg#only-dark){ .twemoji } [Ente](photo-management.md#ente)
+- ![Stingle logo](assets/img/photo-management/stingle.png#only-light){ .twemoji }![Stingle logo](assets/img/photo-management/stingle-dark.png#only-dark){ .twemoji } [Stingle](photo-management.md#stingle)
+- ![PhotoPrism logo](assets/img/photo-management/photoprism.svg){ .twemoji } [PhotoPrism](photo-management.md#photoprism)
+
+</div>
+
```
Comment (jonaharagon) [inline docs/tools.md:182]:
```suggestion

[Learn more :material-arrow-right-drop-circle:](photo-management.md)

```

File: docs/photo-management.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    ![Stingle logo](assets/img/photo-management/stingle.png#only-light){ align=right }
+    ![Stingle logo](assets/img/photo-management/stingle-dark.png#only-dark){ align=right }
+
+    **Stingle** is an end-to-end encrypted photo backup service. It includes a gallery and camera application which can directly upload to the service. Storage starts at 1GB for free accounts.
+
+    [:octicons-home-16: Homepage](https://stingle.org/){ .md-button .md-button--primary }
+    [:octicons-eye-16:](https://stingle.org/privacy/){ .card-link title="Privacy Policy" }
+    [:octicons-info-16:](https://stingle.org/faq/){ .card-link title=Documentation}
+    [:octicons-code-16:](https://github.com/ente-io){ .card-link title="Source Code" }
```
Comment (jonaharagon) [inline docs/photo-management.md:46]:
```suggestion
    **Stingle** is a gallery and camera application with built-in, end-to-end encrypted backup and sync functionality for your photos and videos. Storage starts at 1GB for free accounts on their cloud, or you can host your own Stingle API server for total independence.

    [:octicons-home-16: Homepage](https://stingle.org/){ .md-button .md-button--primary }
    [:octicons-eye-16:](https://stingle.org/privacy/){ .card-link title="Privacy Policy" }
    [:octicons-info-16:](https://stingle.org/faq/){ .card-link title=Documentation}
    [:octicons-code-16:](https://github.com/stingle){ .card-link title="Source Code" }
```

**Review by jonaharagon — COMMENTED** [review]
@dngray

**Comment by dngray** [comment]
I think this one is good to go.

**Review by dngray — APPROVED** [review]

**Review by freddy-m — APPROVED** [review]

**Comment by aviv926** [comment]
In the announcement on 3.18 it was[ written that Immich was added](https://discuss.privacyguides.net/t/v3-18/15135#:~:text=Add%20Photo%20Management,83), but in practice it is not

**Comment by jonaharagon** [comment]
Where are you reading that? As far as I know Immich wasn't one we looked at actually, @dngray did you?

**Comment by aviv926** [comment]
> Where are you reading that? As far as I know Immich wasn't one we looked at actually, @dngray did you?

Does it not meet the requirements?

**Comment by dngray** [comment]
Immich was not the one we looked at, we looked at ente, stingle and photoprism.

**Pull request merged**